mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1444,7 +1451,7 @@ def predict_with_latency(X_copy):
                 i, row_data = row
                 single_input = row_data.to_frame().T if is_dataframe else row_data
                 start_time = time.time()
-                y_pred = self.model_predict_fn(single_input)
+                y_pred = self.predict_fn(single_input)
```

Comment:
@harupy Is there more descriptive naming we can use for `self.predict_fn` and `self.model_predict_fn`? The two sound very similar / potentially interchangeable right now.

Reply:
This PR removed `model_predict_fn`. Having both `self.predict_fn` and `self.model_predict_fn` is confusing and highly-error prone.

Reply:
Excellent - sorry I missed that